meta/saiserialize.c

```diff
@@ -1355,3 +1355,17 @@ int sai_deserialize_attribute(
 
     return (int)(buf - buffer);
 }
+
+int sai_serialize_double(
+        _Out_ char *buffer,
+        _In_ sai_double_t d64)
+{
+    return sprintf(buffer, "%.2lf", d64);
```

Comment:
Throwing off most of significant places make no sense using double use uint64 or int64